core/src/test-core/java/com/radixdlt/targeted/mempool/MempoolFillerUpdate.java

```diff
@@ -64,83 +64,36 @@
 
 package com.radixdlt.targeted.mempool;
 
+import com.radixdlt.consensus.event.LocalEvent;
 import java.util.Objects;
-import java.util.Optional;
-import java.util.OptionalInt;
-import java.util.concurrent.CompletableFuture;
 
 /** An update event to the mempool filler */
-public final class MempoolFillerUpdate {
+public final class MempoolFillerUpdate implements LocalEvent {
   private final int parallelTransactions;
   private final boolean sendToSelf;
-  private final CompletableFuture<Void> completableFuture;
 
-  private MempoolFillerUpdate(
-      int parallelTransactions, boolean sendToSelf, CompletableFuture<Void> completableFuture) {
+  private MempoolFillerUpdate(int parallelTransactions, boolean sendToSelf) {
     this.parallelTransactions = parallelTransactions;
     this.sendToSelf = sendToSelf;
-    this.completableFuture = completableFuture;
   }
 
   public static MempoolFillerUpdate enable(int parallelTransactions, boolean sendToSelf) {
-    return new MempoolFillerUpdate(parallelTransactions, sendToSelf, null);
-  }
-
-  public static MempoolFillerUpdate enable(
```

Comment:
Mempool filler seems no longer used, can we remove it? together with `MempoolFillerStarter` and related messages, etc?

Reply:
Done